TWLight/resources/templates/resources/partner_detail.html

```diff
@@ -219,6 +219,25 @@ <h1>{{ object }}</h1>
         </div>
       </div>      
     </li>
+    {% with object.excerpt_limit as excerpt_limit and object.excerpt_limit_percentage as excerpt_limit_percentage %}
+    {% if excerpt_limit or excerpt_limit_percentage %}
+      <li class="timeline-inverted">
+        <div class="timeline-badge"><i class="fa fa-files-o" aria-hidden="true"></i>
+        </div>
+        <div class="timeline-panel">
+          <div class="timeline-body">
+            {% comment %} Translators: If a partner has specified the excerpt limit, this message will display the number of words or percentage limit on the partner page. {% endcomment %}
+            <p><em>{%trans "Excerpt limit: " %}</em></p>
```

Comment:
Per my comment on https://github.com/WikipediaLibrary/TWLight/pull/123, we shouldn't break up translation strings like this as it makes it hard to translate the full sentence coherently.

You're probably better off writing out the sentence in full for each case.